app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params.dig(:session, :email)&.downcase
+    if user&.authenticate params.dig(:session, :password)
+      reset_session
+      log_in user
+      redirect_to user, status: :see_other
+      flash[:success] = I18n.t "flash.loginSuccess"
```

Comment:
```suggestion
      flash[:success] = t "flash.loginSuccess"
```
viết thế này có chạy được ko em